openedx/core/djangoapps/course_roles/models.py

```diff
@@ -0,0 +1,86 @@
+"""
+Models for course roles schema
+"""
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from organizations.models import Organization
+
+User = get_user_model()
+
+
+class CourseRolesRole(models.Model):
+    """
+    Model for a course role.
+    """
+    class RoleType(models.TextChoices):
+        """
+        Enum for the role type.
+        """
+        DEFAULT = 'default'
```

Comment:
What's the distinction bewteen a DEFAULT role and a CUSTOM role?

Reply:
default role is used for the roles that exist for all contexts and custom role will be used for fully custom roles and for changing the settings for a default role in a specific context (organization).
But custom roles won't be implemented yet, so I'm going to remove this from the code.